packages/table/src/table.tsx

```diff
@@ -435,6 +439,74 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         };
     }
 
+    // Instance methods
```

Comment:
nit: these sort of block comments indicate the file is too big and should be split. "nit" because i know that splitting is extremely difficult at this point.

Reply:
Agree. I want to change this later.